src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -26,6 +28,9 @@
 import java.util.*;
 import java.util.function.Predicate;
 
+import static com.minecolonies.api.util.constant.ToolLevelConstants.TOOL_LEVEL_HAND;
```

Comment:
@Raycoms should I change for  
import static com.minecolonies.api.util.constant.ToolLevelConstants.*; 

as well?